packages/p2p/src/components/advertiser-page/advertiser-page.jsx

```diff
@@ -12,12 +12,15 @@ import UserAvatar from 'Components/user/user-avatar/user-avatar.jsx';
 import { useStores } from 'Stores';
 import AdvertiserPageStats from './advertiser-page-stats.jsx';
 import AdvertiserPageAdverts from './advertiser-page-adverts.jsx';
+import AdvertiserPageDropdown from './advertiser-page-dropdown.jsx';
 import TradeBadge from '../trade-badge/trade-badge.jsx';
 import './advertiser-page.scss';
 
 const AdvertiserPage = () => {
     const { advertiser_page_store, buy_sell_store } = useStores();
 
+    const [is_dropdown_visible, setIsDropdownVisible] = React.useState(false);
```

Comment:
Can we add this to the `advertiser_page_store` so it can be used across files and components without being passed as props.